lib/ruby_lsp/ruby_lsp_rails/runner_client.rb

```diff
@@ -109,7 +109,7 @@ def initialize(outgoing_queue)
           end,
           Thread,
         )
-      rescue Errno::EPIPE, IncompleteMessageError
+      rescue Errno::EPIPE, IncompleteMessageError, MessageError
```

Comment:
(Perhaps we should rescue _any_ `StandardError` here?)

Reply:
That sounds good to me.